dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -532,146 +522,128 @@ class RegistrationsControllerTest < ActionController::TestCase
 
   # USING NEW SIGN-UP FLOW [START]
 
-  test "teachers go to specified return to url after signing up [new sign up flow]" do
-    session[:user_return_to] = user_return_to = '//test.code.org/the-return-to-url'
-
-    assert_creates(User) do
-      post :create, params: {user:  @default_params_new_signup.update(user_type: 'teacher', age: '21+', email_preference_opt_in: true)}
-    end
-
-    assert_redirected_to user_return_to
-  end
```

Comment:
This test will be added as part of the work in [this ticket](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-2517)